Stores/src/main/java/StoresLocalSettings.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Infomaniak Core - Android
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.lib.stores
+
+import android.content.Context
+import com.infomaniak.lib.core.utils.SharedValues
+import com.infomaniak.lib.core.utils.transaction
+
+class StoresLocalSettings private constructor(context: Context) : SharedValues {
+
+    override val sharedPreferences = context.applicationContext.getSharedPreferences(SHARED_PREFS_NAME, Context.MODE_PRIVATE)!!
```

Comment:
Use a DataStore instead of SharedPreferences.

Reply:
Done in #131